src/mpid/ch4/generic/am/mpidig_am.h

```diff
@@ -7,6 +7,7 @@
 #define MPIDIG_AM_H_INCLUDED
 
 #define MPIDI_AM_HANDLERS_MAX (64)
+#define MPIDI_AM_RNDV_CB_MAX  (10)
```

Comment:
Shouldn't this be a power of 2 since it represents the max rndv CB that can be carried in the flag? We should also have a compiler static check just like the checks against `MPIID_AM_HANDLERS_MAX`.

Reply:
I don't think we need define the MAX to be the maximum range of the bits. Leaving some room affords some flexibility in future to adjust the bits. I agree we can add some assertion checks.